apps/comps/package.json

```diff
@@ -40,8 +40,7 @@
     "react-hook-form": "^7.54.2",
     "viem": "^2.28.0",
     "wagmi": "^2.14.8",
-    "yup": "^1.6.1",
-    "zod": "^3.24.4"
+    "zod": "^3.24.2"
```

Comment:
Tracked down all the weird problems we were having to that you had installed a newer version of Zod in this project than what the other projects were using. The error messages we were seeing were strange and not helpful at all, but I noticed the thing they had in common was some relationship, sometimes very indirectly, to Zod. I made sure all packages use the same Zod version, and things seem to work now.